spec/requests/api/actions_spec.rb

```diff
@@ -68,6 +68,120 @@
       end
     end
 
+    describe 'edge case country names' do
+      it 'successfully posts Bolivia to the queue' do
```

Comment:
There's a lot of repetition in these specs, but I wasn't sure of a better way to handle them that wouldn't ruin error reporting if we were to have a failure. As such, I spelled them all out individually but if someone has a suggestion for how to DRY this up, I'd be happy to hear it.